examples/kubernetes/cilium.yaml

```diff
@@ -1,3 +1,67 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cilium-config
+  namespace: kube-system
+data:
+  # This etcd-config contains the etcd endpoints of your cluster. If you use
+  # TLS please make sure you uncomment the ca-file line and add the respective
+  # certificate has a k8s secret, see explanation on line 38.
+  etcd-config: |-
+    ---
+    endpoints:
+    - http://127.0.0.1:2379
+    #
+    # In case you want to use TLS in etcd, uncomment the following line
+    # and add the certificate in cilium-etcd-secrets on line 38.
```

Comment:
This should refer to a comment / tag instead of a specific line number, as the line number can change if someone edits this file.